profileutil/plist_data.go

```diff
@@ -98,6 +101,25 @@ func (profile PlistData) GetExportMethod() exportoptions.Method {
 	return exportoptions.MethodDefault
 }
 
+// GetExportMethodMac ...
+func (profile PlistData) GetExportMethodMac() exportoptions.Method {
```

Comment:
merge this with GetExportMethod -> `GetExportMethod(profileType ...ProfileType)`